ros/src/tl_detector/tl_detector.py

```diff
@@ -124,12 +124,16 @@ def get_light_state(self):
             x2 = box[1][0]
             y2 = box[1][1]
             tl_image = cv_image[y1:y2, x1:x2]
-            #classifier_size = (128,128)
-            #resized = cv2.resize(tl_image, classifier_size, cv2.INTER_LINEAR)
-            resized = tl_image
+
+            # Skip small images to avoid false positives
+            min_img_height = 50
+            if tl_image.shape[0] < min_img_height:
```

Comment:
Wouldn't it be better to adjust this in https://github.com/Kairos-Automotive/carla-brain/blob/75c95f5eb88cc83c1411c9f524e9d8c3936cb4f6/ros/src/tl_detector/tl_detector_segmentation.py#L97 
Just to have one location filtering out the detected tl.

Reply:
good question.
there I resize the image to 384x288 before detection. if original size was 800x600 then the condition above is roughly w<8, h<16, in terms of original pixels

Reply:
i suggest we merge all PRs in. i will then look at tl_detector cleanup/adjustment, based on ros bags